lib/vanagon/platform/rpm.rb

```diff
@@ -45,15 +45,15 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}.#{project.noarch ? 'noarch' : @architecture}.rpm"
       end
 
-      def output_dir(target_repo = "products")
-        super
-      end
```

Comment:
oh yes, for sure! This was mostly a note so future us didn't forget we broke this last time :)